Kitodo/src/test/java/org/kitodo/production/metadata/MetadataEditorIT.java

```diff
@@ -115,7 +128,9 @@ public void shouldAddMultipleStructuresWithMetadataGroup() throws Exception {
             Locale.LanguageRange.parse("en"));
         String metadataKey = "TitleDocMain";
 
-        File metaXmlFile = new File("src/test/resources/metadata/2/meta.xml");
+        int testProcessId = MockDatabase.insertTestProcess(TEST_PROCESS_TITLE, 1, 1, 1);
+        ProcessTestUtils.copyTestMetadataFile(testProcessId, TEST_METADATA_FILE);
+        File metaXmlFile = new File("src/test/resources/metadata/" + testProcessId + "/meta.xml");
```

Comment:
It would be good if the path or parts of the path could be generalised and reused.